eureka2-write-server/src/main/java/com/netflix/eureka2/server/transport/tcp/registration/TcpRegistrationHandler.java

```diff
@@ -32,25 +33,23 @@
 import rx.Observable;
 import rx.schedulers.Schedulers;
 
-import javax.inject.Inject;
-
 /**
  * @author Tomasz Bak
  */
 public class TcpRegistrationHandler implements ConnectionHandler<Object, Object> {
 
     private final WriteServerConfig config;
-    private final SourcedEurekaRegistry<InstanceInfo> registry;
+    private final EurekaRegistrationProcessor<InstanceInfo> registrationProcessor;
     private final EvictionQueue evictionQueue;
     private final WriteServerMetricFactory metricFactory;
 
     @Inject
     public TcpRegistrationHandler(WriteServerConfig config,
-                                  SourcedEurekaRegistry registry,
+                                  @Named("registration") EurekaRegistrationProcessor registrationProcessor,
```

Comment:
Can we use the fields in the com.netflix.eureka2.Names class here instead of strings? (Also the same in the Module binding).